packages/accounts/package.json

```diff
@@ -39,7 +39,7 @@
     "test:run-e2e": "vitest run --config vitest.config.e2e.ts"
   },
   "devDependencies": {
-    "@alchemy/aa-core": "^0.1.0",
+    "@alchemy/aa-core": "^0.1.1",
```

Comment:
you shouldn't have to manually update this actually

Lerna will do this for you when we publish a new package